### PR TITLE
Upgarde k8s packages

### PR DESCRIPTION
**Pull request by prankulmahajan**

### CODE REVIEW

**Review by sameshai — APPROVED** [review]
LGTM

File: pkg/mountmanager/mount_linux.go [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+package mountmanager
+
+import (
+	"os"
+
+	mount "k8s.io/mount-utils"
+)
+
+// MakeFile creates an empty file.
+func (m *NodeMounter) MakeFile(path string) error {
```
Comment (prankulmahajan) [inline pkg/mountmanager/mount_linux.go:29]: @sameshai Earlier in k8s v1.14, these functions were part of the mount package, https://github.com/kubernetes/kubernetes/blob/release-1.14/pkg/util/mount/mount_linux.go#L389-L412

Comment (sameshai) [reply pkg/mountmanager/mount_linux.go:29]: Ok got it

**Review by arahamad — APPROVED** [review]
Please verify

**Pull request merged**